src/cli/flags/options.go

```diff
@@ -16,7 +16,7 @@ const (
 	FetchParallelismFN          = "fetch-parallelism"
 	NoStatsFN                   = "no-stats"
 	RecoveredErrorsFN           = "recovered-errors"
-	RestorePermissionsFN        = "restore-permissions"
+	SkipPermissionsFN           = "skip-permissions"
```

Comment:
nit: `no` would be clearer than `skip` here.
```suggestion
	NoPermissionsFN           = "no-permissions"
```